meta-oe/recipes-support/tio/tio_2.0.bb

```diff
@@ -0,0 +1,3 @@
+require tio.inc
+SRC_URI = "git://github.com/tio/tio;protocol=https;nobranch=1"
```

Comment:
why `nobranch`

Reply:
So that the recipe still works, if someone renames the default branch to main or something else. If it's not wanted, i could change that too.